Resources/lang/fr/common.php

```diff
@@ -45,7 +45,7 @@
     'rank'      => 'Grade',
     'ranks'     => 'Grades',
     'record'    => 'Récapitulatif',
-    'reports'   => 'Rapports de vols',
+    'reports'   => 'Rapports de pilote (PIREP)',
```

Comment:
Hm, "Rapport de vols" is a report about a flight while "Rapports de pilote" is a report about a pilot. I don't know what you think but "Rapports de vols" may be better since a PIREP is more about a flight flown by someone than someone